website/docs/general/blog-contributing-guide.md

```diff
@@ -67,16 +81,16 @@ keywords:
 - keywords 3
 - keywords 4
 - keywords 5
-description: description of this blog
+description: Description of the post
 tags: [tag1,tag2,...,tagn]
 ```
 
 ##### Co-author template
 
-[Co-author](https://docusaurus.io/docs/next/blog#blog-post-authors) feature is added, since translating and editing articles is also time-consuming, we would like to give credit to translators and technical writers as well.
+Translating and editing articles consume time and effort and deserves credit. For this, you can use the co-author template to add multiple authors to the blog post.
```

Comment:
What do you think of combining it with the last section itself, it kind of looks repeated?

Reply:
I think we can keep them separate for now. With the doc revamping in the pipeline, we can decide how best to structure it.